src/Windows/local_cache.cpp

```diff
@@ -70,7 +70,7 @@ static void init_callback()
     } 
     else if (wenv_home != L"" && wenv_home[0] != 0)
     {
-        dirname = wenv_home;
+        dirname = wenv_home.append(L"..\\..\\LocalLow");
```

Comment:
My %LOCALAPPDATA% is "C:\Users\yenlee.REDMOND\AppData\Local". Is the intended dirname "C:\Users\yenlee.REDMOND\LocalLow"?

Reply:
![Capture](https://user-images.githubusercontent.com/58270794/80149071-d84a5880-856a-11ea-8392-5fe9d012272f.PNG)